#File containing parameters for the different tasks

move_all:
  max_stable_counts: 15
  yaw_threshold: 0.1
  depth_threshold: 0.1

dive:
  stablePeriods: 50


lanes:
  foundCounts: 10
  centerStableCounts: 60
  centerMaxError: 2.0
  turnStableCount: 30
  turnMaxError: 0.1


roulette:
  foundCounts: 15
  centerStableCounts: 150
  centerMaxError: 3.0

dice:
  foundCounts: 100
  centerStableCounts: 50
  centerMaxError: 0.4

vacuum:
  suckDuration: 100
  suckingPower: 400

roll:
  rollDuration: 20
